nistats/reporting/glm_reporter_templates/report_template.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, inital-scale=1, shrink-to-fit=yes">
+    <meta charset="utf-8">
+    <title>${page_title}</title> <!-- func:glm_reporter._make_headings() -->
+
+    <style>
+        html {
+            font-family: sans-serif;
+        }
+
+        body {
+            text-align: center;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        table, th, td {
+            border: 1px solid black;
+            border-collapse: collapse;
+            margin-left: auto;
+            margin-right: auto;
+            padding: 5px;
+            padding-left: 15px;
+            padding-right: 15px;
+            text-align: center;
+        }
+
+        tr, td {
+            border-bottom: 0px;
+            border-top: 0px;
+        }
+
+        tr:nth-child(even) {
+            background-color: lightgray;
+        }
+
+        tr:hover {
+            background-color: yellow;
+        }
+
+        details:hover {
+            cursor: pointer;
+        }
+        details > summary {
+            margin-left: auto;
+            margin-right: auto;
+            width: 25%;
+            background-color: lightgray;
+            outline-color: lightgray;
+        }
+
+        address {
+            color: lightgray;
+        }
+        address > a:visited {
+            color: lightgray;
+        }
+    </style>
+
+</head>
+    <body>
+
+        <h1>${page_heading_1}</h1>  <!-- func:glm_reporter._make_headings() -->
+        <h2>${page_heading_2}</h2>  <!-- func:glm_reporter._make_headings() -->
+
+        <h3>Model details:</h3>
+        ${model_attributes}  <!-- func:glm_reporter._model_attributes_to_dataframe() -->
+
+        <h3>Design Matrix:</h3>
+        ${design_matrices}  <!-- func:glm_reporter._dmtx_to_svg_url() -->
+
+        <h3 id="all-contrasts">Contrasts</h3>
+        ${all_contrasts_with_plots}  <!-- func:glm_reporter._plot_contrasts() -->
+
+        <h3>Mask</h3>  <!-- func:glm_reporter._mask_to_svg() -->
+        <img src="data:image/svg+xml,${roi_plot}"
+             alt="Pass the mask with the `roi_img` parameter to plot the ROI"/>
+
+        <h3> Stat Maps with Cluster Tables</h3>
+        ${component}  <!-- func:glm_reporter._make_stat_maps_contrast_clusters() -->
+        <br>
+    </body>
+    <address>
```

Comment:
should not be outside of the `<body>` . please check the generated reports for other errors e.g. using

https://validator.w3.org/